suite/runsuite_wrapper.pl

```diff
@@ -264,6 +306,63 @@
+                'prof_pcs,thread_private|common.broadfun' => 1, # i#4845
+                'prof_pcs,thread_private|common.fib' => 1, # i#4845
+                'prof_pcs,thread_private|common.decode-bad' => 1, # i#4845
+                'prof_pcs,thread_private|common.decode' => 1, # i#4845
+                'prof_pcs,thread_private|common.decode-stress' => 1, # i#4845
+                'prof_pcs,thread_private|common.eflags' => 1, # i#4845
+                'prof_pcs,thread_private|common.floatpc' => 1, # i#4845
+                'prof_pcs,thread_private|common.floatpc_xl8all' => 1, # i#4845
+                'prof_pcs,thread_private|common.getretaddr' => 1, # i#4845
+                'prof_pcs,thread_private|common.nativeexec' => 1, # i#4845
+                'prof_pcs,thread_private|common.protect-dstack' => 1, # i#4845
+                'prof_pcs,thread_private|common.segfault' => 1, # i#4845
+                'code_api,loglevel|common.nativeexec' => 1, # i#4845
+                'code_api,stack_size|common.nativeexec' => 1, # i#4845
+                'enable_full_api|common.nativeexec' => 1, # i#4845
+                'code_api,stack_size,loglevel,no_hide|common.nativeexec' => 1, # i#4845
+                '|common.nativeexec' => 1, # i#4845
+                'code_api,tracedump_text,tracedump_origins|common.nativeexec' => 1, # i#4845
+                'code_api,tracedump_text,tracedump_origins,syntax_intel|common.nativeexec' => 1, # i#4845
+                'code_api,thread_private,tracedump_binary|common.nativeexec' => 1, # i#4845
+                'code_api,bbdump_tags|common.nativeexec' => 1, # i#4845
+                'checklevel|common.nativeexec' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.broadfun' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.logstderr' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.fib' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.decode-bad' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.decode' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.decode-stress' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.eflags' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.floatpc' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.floatpc_xl8all' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.getretaddr' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.nativeexec' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.protect-dstack' => 1, # i#4845
+                'desktop,coarse_freeze_at_exit|common.segfault' => 1, # i#4845
+                'finite_shared_bb_cache,cache_shared_bb_regen|common.nativeexec' => 1, # i#4845
+                'finite_shared_trace_cache,cache_shared_trace_regen|common.nativeexec' => 1, # i#4845
                 );
             $issue_no = "#2145";
```

Comment:
I don't think there's a way to map each test with it's own issue number, and IMHO there isn't a strong need either. Let me know though if you think otherwise.

Reply:
This is fine.